components/Tabs/src/private/MDCItemBarCell.m

```diff
@@ -260,12 +271,11 @@ - (void)layoutSubviews {
       titleCenter.y = roundf((float)titleCenter.y);
 
     } else {
-      // Title only, center vertically.
-      // +1 for slight adjustment to font baseline for centered title labels.
-      const CGFloat centeredTitleYOffset = 1.0f;
-      titleCenter.y = contentBounds.size.height - kBottomPadding - titleSize.height +
-                      centeredTitleYOffset + ((float)titleSize.height / 2.0f);
-      titleCenter.y = roundf((float)titleCenter.y);
+      CGFloat scale = self.window.contentScaleFactor;
```

Comment:
Why we pull in the scale factor here is not obvious and threw me for a loop - add comment? Why do we need to do this special rounding here and not most other places?

Reply:
We should be doing it everywhere we compute the frame/center/bounds of a view - images and text are the real important ones.  In this case, I'm aligning the center to avoid introducing errors, but I'm not aligning the bounds since it isn't part of this issue. During testing, I didn't see any misaligned tabs.